cypress/features/regression/designSystem/menu.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Design Systems Menu component
+  I want to test the Design Systems Menu component
+    
+  @positive
+  Scenario Outline: Scenario name: Test that all child link elements are part of a scrollable block
```

Comment:
Please delete this - probably left when you've created a step with auto completion - `Scenario name:`